### ISSUE TITLE
should optimize static files for the production deployment

### ISSUE THREAD

**Issue by hoatle** [HumanityOnline/MyRefuge#13]
currently, the static files are served by heroku, it should be served by amazon s3.

maybe we should unuse whitenoice, rednoise on production.

note: need to check on media files configuration on production.

**Comment by cazgp** [comment]
Depends what you mean by "optimise" if you mean "increase costs for no real necessary reason", then sure :P Putting them behind a CDN is the better alternative that usually works out cheaper. CloudFront is one possible alternative seeing as there is an Amazon account.

**Comment by hoatle** [comment]
yes, this should work: http://whitenoise.readthedocs.org/en/latest/django.html#use-a-content-delivery-network-optional
